### PR TITLE
Added openvpn-status script

### PR DESCRIPTION
**Pull request by Vincevrp**
This script shows the running openvpn config file.

### CODE REVIEW

**Comment by Vincevrp** [comment]
Depends, maybe some people prefer not to show the filename? I would keep `isrunning-openvpn` as a simpler alternative.

**Comment by x70b1** [comment]
Then we could rename your script to `openvpn-filename` and the other one to `openvpn-isrunning`?

**Comment by Vincevrp** [comment]
Maybe `openvpn-config`?

**Comment by x70b1** [comment]
Probably doesn't matter for now. If we notice later that the name is not the best it can be renamed again. This already happened with some scripts.

We can call it `openvpn-status`. Then we can simply add more features when we have them.

**Comment by Vincevrp** [comment]
Alright!

File: polybar-scripts/openvpn-status/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Script: openvpn-status
+
+Show the running openvpn config file
+
+![VPN down](screenshots/down.png)
```
Comment (x70b1) [inline polybar-scripts/openvpn-status/README.md]: Please rename it to `1.png` and `2.png`

File: polybar-scripts/openvpn-status/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+![VPN connected](screenshots/connected.png)
+
+## Dependencies
```
Comment (x70b1) [inline polybar-scripts/openvpn-status/README.md]: I guess we can delete this. 
Everyone should know that he need OpenVPN ifhe want to use this script.

File: polybar-scripts/openvpn-status/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+```
+openvpn --daemon --auth-nocache --cd "/etc/openvpn/conf.d" --config "UK-Southampton.ovpn"
+```
+
+## Module
+
+```ini
+[module/openvpn]
+type = custom/script
+label = VPN: %output%
```
Comment (x70b1) [inline polybar-scripts/openvpn-status/README.md]: The `VPN:` text part should be part of the script.

File: polybar-scripts/openvpn-status/openvpn-status.sh [hunk 4]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```
Comment (x70b1) [inline polybar-scripts/openvpn-status/openvpn-status.sh]: please use `#!/bin/sh` to be independent from `bash`

**Comment by Vincevrp** [comment]
Fixed it!

**Pull request merged**